examples/app_commands/basic.py

```diff
@@ -68,4 +68,30 @@ async def joined(interaction: discord.Interaction, member: Optional[discord.Memb
     await interaction.response.send_message(f'{member} joined in {member.joined_at}')
 
 
+# A Context Menu is an app command that can be run on a member or on a message by rightclicking.
+# It always takes an interaction as its first parameter and a Member, Message or Union of both as its second parameter.
+
+# This context menu only works for members
+@client.tree.context_menu(name='Joindate')
+async def show_join_date(interaction: discord.Interaction, member: discord.Member):
+    await interaction.response.send_message(f'{member} joined in {member.joined_at}')
+
+
+# This context menu only works for messages
+@client.tree.context_menu(name='Report to mods')
```

Comment:
```suggestion
@client.tree.context_menu(name='Report to Moderators')
```

Sticking with title case here.